src/test/java/net/openhft/chronicle/queue/CheckHalfWrittenMsgNotSeenByTailerTest.java

```diff
@@ -79,6 +80,11 @@ private static void writeIncompleteMessage(String arg, boolean exit) throws Inte
         }
     }
 
+    @Before
+    public void setUp() {
+        ignoreException("Forced unlocking `chronicle.write.lock` in lock");
+    }
```

Comment:
Not sure why this was necessary, but this test appeared to fail with it occasionally, which seems expected given what it's doing.